ansible-ipi-install/roles/node-prep/tasks/10_validation.yml

```diff
@@ -519,16 +519,6 @@
         msg: 'Adding hosts to dynamic HP group failed or redfish_inspection was set to false. All inventory systems will use IPMI.'
   tags: validation
 
-- name: Fail when provisioningHostIP and bootstrapProvisioningIP are not set when virtualmedia option is enabled
```

Comment:
I've removed this validation because for newer versions this is not needed:

https://github.com/openshift-kni/baremetal-deploy/blob/ef54d0d1b0a3c6f8830ad04e8076afec3effb12d/documentation/ipi-install/modules/ipi-install-modifying-install-config-for-no-provisioning-network.adoc#L29-L36

I could re-add it with a conditional, but since 4.6 is already EOL I was wondering if it was worth doing it. 

https://github.com/openshift-kni/baremetal-deploy/blob/ef54d0d1b0a3c6f8830ad04e8076afec3effb12d/documentation/ipi-install/modules/ipi-install-modifying-install-config-for-no-provisioning-network.adoc#L11-L21

At this point the latest supported version is 4.10[1] , wdyt?

[1] https://access.redhat.com/support/policy/updates/openshift

Reply:
In the end decided to keep the validation but only for the documented version (4.6) other versions do not require the `provisioningHostIP`